### PR TITLE
✨ Add getting original image url

### PR DESCRIPTION
**Pull request by PetrPol**
Added option to create original image url,
to load original image without any processing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.3] - 2021-04-13
```
Comment (davidbilik) [inline CHANGELOG.md]: 2.1.0 + date is wrong

File: sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt [hunk 2]
```diff
@@ -232,11 +244,15 @@ class Resizin(private val appId: String) {
...
-                append("image").append("/")
-                val transformations = createTransformations()
-                append(transformations.filter { it.isNotEmpty() }.joinToString("-"))
-                if (transformations.isNotEmpty()) {
-                    append("/")
+                if (useOriginalImage) {
+                    append("original").append("/")
+                } else {
+                    append("image").append("/")
+                    val transformations = createTransformations()
```
Comment (davidbilik) [inline sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt]: I'd probably create the transformations beforehand and check 
```
if (useOriginalImage && transformations.isNotEmpty()) throw IllegalStateException("You cannot specify transformations and use original image")
```

File: CHANGELOG.md (hunk 1 above)
Comment (PetrPol) [reply CHANGELOG.md]: Updated to version 2.1.0 and updated date: e4834a9e53eaba33608dfad3715b2bffc615bca5

File: sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt (hunk 2 above)
Comment (PetrPol) [reply sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt]: Added check to original and transformation at the same time: a6df63d11aa811445d2e32f835152b9ad4919f04

Comment (davidbilik) [reply sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt]: it's sad that you forgot the formatting rules we apply in Ackee :D please add curly braces around the if body, thanks

Comment (PetrPol) [reply sdk/src/main/java/io/github/ackeecz/resizin/sdk/Resizin.kt]: I don't forgot, I'm just.. trouba :D sorry 44a638ffa3348cea93ee05e1227e108f29ddebb0

**Pull request merged**